website/docs/premake_OS.md

```diff
@@ -2,6 +2,10 @@
 id: _OS
 ---
 
+:::caution_
+**This has been deprecated in Premake 5.0 alpha 12.** Use the new [_TARGET_OS](premake_TARGET_OS.md) instead.
```

Comment:
Is this really deprecated that far back, or is it deprecated at the current version?

Reply:
This commit introduces the change, and was first included in alpha 12. 5f589ad5